docs/configure.rst

```diff
@@ -0,0 +1,113 @@
+.. highlight:: console
+.. _configure:
+
+
+=========
+Configure
+=========
+
+
+Install Croud CLI
```

Comment:
```suggestion
Install Croud
```

the intro para should use "Croud CLI" but from then on we can just call it Croud